js_modules/dagit/packages/core/src/ticks/tests/EvaluateSchedule.mocks.tsx

```diff
@@ -0,0 +1,170 @@
+import {MockedResponse} from '@apollo/client/testing';
+
+import {GET_SCHEDULE_QUERY, SCHEDULE_DRY_RUN_MUTATION} from '../EvaluateSchedule';
+import {GetScheduleQuery, ScheduleDryRunMutation} from '../types/EvaluateSchedule.types';
+
+export const GetScheduleQueryMock: MockedResponse<GetScheduleQuery> = {
+  request: {
+    query: GET_SCHEDULE_QUERY,
+    variables: {
+      scheduleSelector: {
+        scheduleName: 'test',
+        repositoryLocationName: 'testLocation',
+        repositoryName: 'testName',
+      },
+    },
+  },
+  result: {
+    data: {
+      __typename: 'DagitQuery',
+      scheduleOrError: {
+        __typename: 'Schedule',
+        name: 'configurable_job_schedule',
+        potentialTickTimestamps: [1, 2, 4, 5, 6, 7, 8, 9, 10],
+      },
+    },
+  },
+};
+
+export const scheduleDryWithWithRunRequest = {
```

Comment:
I think if you put the `ScheduleDryRunMutation` type on this line you can avoid all the inline const annotations?

Reply:
If I do that then it's going to think the fields in the object are nullable since they are nullable in the `ScheduleDryRunMutation` type and I want to use this object in my test to iterate over the run requests